src/main/java/org/alliancegenome/curation_api/jobs/executors/Gff3Executor.java

```diff
@@ -182,24 +208,77 @@
 		return idsAdded;
 	}
 
-	private Map<String, List<Long>> loadAssociations(BulkLoadFileHistory history, List<Gff3DTO> gffData, Map<String, List<Long>> idsAdded,
-			BackendBulkDataProvider dataProvider, String assemblyId, ProcessDisplayHelper ph) {
-		for (Gff3DTO gff3Entry : gffData) {
+	private Map<String, List<Long>> loadLocationAssociations(BulkLoadFileHistory history, List<ImmutablePair<Gff3DTO, Map<String,String>>> gffData, Map<String, List<Long>> idsAdded,
+			BackendBulkDataProvider dataProvider, String assemblyId, Map<String, String> geneIdCurieMap, ProcessDisplayHelper ph) {
+		
+		for (ImmutablePair<Gff3DTO, Map<String,String>> gff3EntryPair : gffData) {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>',' is not followed by whitespace.
<!-- __reviewdog__:ChBlYTUzNTMzOWNjYjhlMjRkEgpjaGVja3N0eWxl -->